### ISSUE TITLE
Unhandled error in plugin series: expected string or bytes-like object

### ISSUE THREAD

**Issue by X-dark** [Flexget/Flexget#2058]
### Expected behaviour:
Plugin series should properly parse qualities.

### Actual behaviour:
Crash at each run

### Steps to reproduce:
- Step 1: ...

#### Config:
https://pastebin.cedricgirard.fr/view/15f3af31
  
#### Log:
https://pastebin.cedricgirard.fr/view/f8c4b3c9

### Additional information:

- FlexGet version: 2.11.24
- Python version: 3.6.4
- Installation method: Distrib package built from git
- Using daemon (yes/no): yes
- OS and version: ArchLinux (Rolling, Up to Date)

**Comment by X-dark** [comment]
It seems I am able to avoid the crash with

```diff
--- /home/cedric/ArchLinux/pkgbuild/flexget-git/src/Flexget/flexget/utils/qualities.py  2018-01-12 13:35:19.877767841 +0100
+++ /usr/lib/python3.6/site-packages/flexget/utils/qualities.py 2018-01-15 18:08:59.442420460 +0100
@@ -41,14 +41,16 @@
         :param string text: data te be tested against
         :returns: tuple (matches, remaining text without quality data)
         """
-
-        match = self.regexp.search(text)
-        if not match:
+        if not isinstance(text,str):
             return False, ""
         else:
-            # remove matching part from the text
-            text = text[:match.start()] + text[match.end():]
-        return True, text
+            match = self.regexp.search(text)
+            if not match:
+                return False, ""
+            else:
+                # remove matching part from the text
+                text = text[:match.start()] + text[match.end():]
+            return True, text
```

**Comment by X-dark** [comment]
This is related to sqlalchemy >= 1.2. Downgrading sqlalchemy python package fix this (without my previous patch).

**Comment by liiight** [comment]
Interesting, thanks for updating.

**Comment by chrisbrasington** [comment]
Failure incurred with installed: community/python-sqlalchemy 1.2.1-1 

Fixed by manuall installing SQLAlchemy 1.1.14 (https://pypi.python.org/pypi/SQLAlchemy/1.1.14) like this..

pip2.7 install https://pypi.python.org/packages/a3/d1/eacdedd964ad43fbc4cc7110dac33fbf1dbd7bf50dedb4dc1799c42972c1/SQLAlchemy-1.1.14.tar.gz
